.github/workflows/performance-tests.yml

```
@@ -0,0 +1,307 @@
+          DEVICE_MODEL: Pixel 6a Emulator
+          OS_VERSION: Android 13
+          BUILD_HASH: ${{ github.sha }}
+          DATASET_SIZE: 1000
+        run: |
+          maestro test .maestro/community/scroll-performance.yaml \
+            --format junit \
+            --output maestro-results/performance-results.xml
+
+      - name: Stop Perfetto Trace Collection
+        if: always()
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2046:warning:2:8: Quote this to prevent word splitting

(shellcheck)